packages/lib/shim-init-node.js

```diff
@@ -186,36 +186,37 @@ function shimInit(options = null) {
 			const nativeImage = require('electron').nativeImage;
 			let image = nativeImage.createFromPath(filePath);
 			if (image.isEmpty()) throw new Error(`Image is invalid or does not exist: ${filePath}`);
-
 			const size = image.getSize();
 
-			let mustResize = size.width > maxDim || size.height > maxDim;
+			const canResize = size.width > maxDim || size.height > maxDim;
 
-			if (mustResize && resizeLargeImages === 'ask') {
-				const answer = shim.showMessageBox(_('You are about to attach a large image (%dx%d pixels). Would you like to resize it down to %d pixels before attaching it?', size.width, size.height, maxDim), {
+			let userAnswer;
+			const YES = 0, NO = 1, CANCEL = 2;
```

Comment:
I use the consts because I thought it'd be confusing having 0, 1 and 2 "loose in the code".

Reply:
Good choice!

Reply:
For naming constants, please check coding_style.md. We don't do uppercase constants, it should be Yes No Cancel